src/commands/Giveaway/gcreate.ts

```diff
@@ -3,36 +3,37 @@ import { SkyraCommand } from '#lib/structures';
 import type { GuildMessage } from '#lib/types';
 import { Schedules } from '#lib/types/Enums';
 import { ApplyOptions } from '@sapphire/decorators';
+import { Args } from '@sapphire/framework';
+import { Time } from '@sapphire/time-utilities';
 import type { TextChannel } from 'discord.js';
 
-const YEAR = 1000 * 60 * 60 * 24 * 365;
+const kWinnersArgRegex = /^([1-9]|\d\d+)w$/i;
+const options = ['winners'];
 
 @ApplyOptions<SkyraCommand.Options>({
 	aliases: ['giveawayschedule', 'gs', 'gc', 'gschedule'],
 	description: LanguageKeys.Commands.Giveaway.GiveawayScheduleDescription,
 	extendedHelp: LanguageKeys.Commands.Giveaway.GiveawayScheduleExtended,
-	requiredPermissions: ['EMBED_LINKS', 'ADD_REACTIONS', 'READ_MESSAGE_HISTORY'],
+	permissions: ['EMBED_LINKS', 'ADD_REACTIONS', 'READ_MESSAGE_HISTORY'],
 	runIn: ['text'],
-	usage: '[channel:textchannelname{2}] <schedule:time> <duration:time> <title:...string{,256}>',
-	flagSupport: true,
-	usageDelim: ' ',
-	promptLimit: Infinity
+	strategyOptions: { options }
 })
-export default class extends SkyraCommand {
-	public async run(
-		message: GuildMessage,
-		[channel = message.channel as TextChannel, schedule, duration, title]: [TextChannel, Date, Date, string]
-	) {
+export class UserCommand extends SkyraCommand {
+	public async run(message: GuildMessage, args: SkyraCommand.Args) {
+		const channel = await args.pick('textChannelName').catch(() => message.channel as TextChannel);
+		const schedule = await args.pick('time');
+		const duration = await args.pick('time');
+		let winners = await args.pick(UserCommand.winners).catch(() => parseInt(args.getOption('winners') ?? '1', 10));
```

Comment:
parseInt is.. crappy, as we've both talked about. Might wanna Number() it? Idk, up to you